src/rws_interface.cpp

```diff
@@ -1003,6 +1008,29 @@ float RWSInterface::getAnalogSignal(std::string const& signal_name)
   return std::stof(getIOSignal(signal_name));
 }
 
+
+IOSignalInfo RWSInterface::getIOSignals()
+{
+  auto const doc = rws_client_.getIOSignals();
+  IOSignalInfo signals;
+  
+  for (auto&& node : xmlFindNodes(doc, {"class", "ios-signal-li"}))
+  {
+    std::string const name = xmlFindTextContent(node, {"class", "name"});
+    std::string const value = xmlFindTextContent(node, {"class", "lvalue"});
+    std::string const type = xmlFindTextContent(node, {"class", "type"});
+    
+    if (!name.empty() && !value.empty())
+      if (type == "DI" || type == "DO")
+        signals[name] = digitalSignalToBool(value);
+      else if (type == "AI" || type == "AO")
+        signals[name] = std::stof(value);
```

Comment:
Add brackets {} to first `if` to separate it more clearly from the second level of `if, else if`.

Reply:
done